setup.cfg

```diff
@@ -39,3 +38,5 @@ testing =
     pytest >= 7.1.1
     pytest-cov >= 3.0.0
     tox >= 3.24.5
+viz =
```

Comment:
@Myoldmopar should we prefer `extras` instead of `viz`?

Reply:
I think viz is good.  It allows the freedom to later add another separate optional install configuration if that's ever needed.